### PR TITLE
Fix: WOW can be used in transfer immediately after switch wallet

### PR DESCRIPTION
**Pull request by egor0798**
Close send asset modal if active wallet is WOW

### CODE REVIEW

**Comment by egor0798** [comment]
Task linked: [CU-86937yycc Transfers. WOW can be used in transfer immediately after switch wallet](https://app.clickup.com/t/86937yycc)

File: src/renderer/entities/asset/ui/AssetCard/AssetCard.tsx [hunk 1]
```diff
@@ -71,7 +72,7 @@ export const AssetCard = ({ chainId, asset, balance, canMakeActions }: Props) =>
             </div>
           )}
         </div>
-        {canMakeActions && (
+        {!walletUtils.isWatchOnly(activeWallet) && (
```
Comment (Asmadek) [inline src/renderer/entities/asset/ui/AssetCard/AssetCard.tsx:75]: You can create variable in model, like `$activeWallet` or `$activeAccounts` but they have different styles and better to ask @tuul-wq which one is correct

**Review by Asmadek — APPROVED** [review]

File: src/renderer/entities/asset/ui/AssetCard/AssetCard.tsx (hunk 1 above)
Comment (tuul-wq) [reply src/renderer/entities/asset/ui/AssetCard/AssetCard.tsx:75]: We can use local model variable when effector model will be implemented for `Assets` page
For now it's absolutely ok to use domain function, especially without props drill 👍

File: src/renderer/widgets/SendAssetModal/ui/SendAssetModal.tsx [hunk 2]
```diff
@@ -56,6 +57,12 @@ export const SendAssetModal = ({ chain, asset }: Props) => {
     priceProviderModel.events.assetsPricesRequested({ includeRates: true });
   }, []);
 
+  useEffect(() => {
```
Comment (tuul-wq) [inline src/renderer/widgets/SendAssetModal/ui/SendAssetModal.tsx:60]: Is it event possible for wallet to change when modal is opened? 🤔

**Review by tuul-wq — APPROVED** [review]

**Pull request merged**